Templates/deletecourse.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Delete Course</title>
+    {% load static %}
+    <link rel="stylesheet" href = "{% static 'styles.css'%}">
+
+    <meta charset="UTF-8">
+</head>
+
+<body>
+  <nav>
+    <div class = "nav-center">
+    <h4>Delete Course</h4>
+      <ul class="nav-links">
+        <li><a href="../home/">Dashboard</a></li>
+        <li><a href="../createcourse/">Create Course</a></li>
+        <li><a href="../editcourse/">Edit Course</a></li>
+      </ul>
+    </div>
+  </nav>
+  
+  <main>
+    <div class="container">
+      <form class="form-control" action="/deletecourse/" method="post">
+      {%  csrf_token %}
+        <table border="1">
+            <tr>
+                <th>ID</th>
+                <th>Name</th>
+                <th>Location</th>
+                <th>Hours</th>
+                <th>Days</th>
+                <th>Instructor</th>
+            </tr>
+          {% for Course in course_list %}
+            <tr>
+                {% if deleted_course.courseID == Course.courseID %}
+                    <td>{{ Course.courseID }}</td>
+                    <td><label type="text" id="coursName" name="coursName" value={{ Course.name }}> </label></td>
+                    <td><label type="text" id="courseLocation" name="courseLocation" value={{ Course.location }}></label></td>
+                    <td><label type="text" id="courseHours" name="courseHours" value={{ Course.hours }}></label></td>
+                    <td><label type="text" id="courseDays" name="courseDays" value={{ Course.days }}></label></td>
+                    <td><label type="text" id="courseInstructor" name="courseInstructor" value={{ Course.instructor }}></label></td>
```

Comment:
What is this part of the code doing? I don't believe getting user input will be necessary if all this is meant to do is delete the course.